modules/mobile/bin/health_checks/request_data/appointments/get_appointments.yaml

```diff
@@ -0,0 +1,26 @@
+sequence:
```

Comment:
These are obviously not intended to be full body checks. I'm just getting a few basic checks up to show that we can test status and some body spot checking.